fits/models.py

```diff
@@ -44,6 +43,15 @@ def __unicode__(self):
         return unicode(self.date_workout)
 
 
+class Weight(models.Model):
+    date_added = models.DateTimeField(auto_now_add=True)
+    owner = models.ForeignKey(User)
+    value = models.FloatField()
```

Comment:
this is value in grams, maybe in kilograms or in stones? Please add unit to name